src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala

```diff
@@ -165,12 +165,30 @@ class BeardTemplateParserSpec extends FunSpec with Matchers {
     describe("attribute identifiers") {
 
       it("should not start with a number") {
-        BeardTemplateParser("{{9hello name=\"Dan\" color = 'blue'}}") should
+        val template = """{{9hello name="Dan" color = 'blue'}}"""
+        BeardTemplateParser(template) should
           be(BeardTemplate(Seq(
             AttrInterpolation(Identifier("hello"), Seq(AttributeWithValue("name", "Dan"), AttributeWithValue("color", "blue"))
             ))))
       }
 
+      it("should not start with a number in multiline") {
+        val template =
+          """<hello>
+            |{{9121hello name="Dan" color = 'blue'}}
+            |</hello>""".stripMargin
+
+        BeardTemplateParser(template) should
+          be(BeardTemplate(Seq(
+            Text("<hello>"),
+            NewLine(1),
+            AttrInterpolation(
+              Identifier("hello"),
+              Seq(AttributeWithValue("name", "Dan"), AttributeWithValue("color", "blue"))),
+            NewLine(1),
+            Text("</hello>"))))
+      }
+
       //        it("should not allow special chars inside of attribute identifiers") {
```

Comment:
It's a little bit tricky :) The Lexer show the error in the correct way, but we have stacktrace from the Parser. 
```
line 1:3 token recognition error at: '!'
{{h!el!l%o name="Dan" color = 'blue'}}
   ^
      
line 1:6 token recognition error at: '!'
{{h!el!l%o name="Dan" color = 'blue'}}
      ^
      
line 1:7 no viable alternative at input 'ell'

head of empty list
java.util.NoSuchElementException: head of empty list
	at scala.collection.immutable.Nil$.head(List.scala:420)
	at scala.collection.immutable.Nil$.head(List.scala:417)
```